src/Packages/Passport/Runtime/ThirdParty/ImmutableBrowserCore/WindowsWebBrowserClientAdapter.cs

```diff
@@ -50,6 +50,8 @@ public void ExecuteJs(string js)
 
         public void LaunchAuthURL(string url, string? redirectUri)
         {
+            // Log the auth URL for test automation to capture
+            Debug.Log($"PASSPORT_AUTH_URL: {url}");
```

Comment:
Just saw this, should use [PassportLogger](https://github.com/immutable/unity-immutable-sdk/blob/main/src/Packages/Passport/Runtime/Scripts/Private/Core/Logging/PassportLogger.cs) for this? So customers can control the logs.